app/models/kudo.rb

```diff
@@ -2,8 +2,14 @@
 
 class Kudo < ApplicationRecord
   validates :title, :content, :employee_id, :receiver_id, :company_value_id, presence: true
+  after_create :add_point
 
   belongs_to :giver_of_kudo, class_name: 'Employee', foreign_key: 'employee_id', inverse_of: :given_kudos
   belongs_to :receiver_of_kudo, class_name: 'Employee', foreign_key: 'receiver_id', inverse_of: :received_kudos
   belongs_to :company_value, class_name: 'CompanyValue'
+
+  def add_point
```

Comment:
using callbacks is considered to be a short way to trouble because it's very easy to lose control over what's happening and when. Feel free to try to implement it without the callback ✌️

Reply:
I've just realised that this method isn't needed anyway :)